### PR TITLE
OpenAI Improvements

### PR DESCRIPTION
**Pull request by philippfromme**
* improves tool descriptions found in [`elementTemplateHandlerDescriptions.json`](https://github.com/bpmn-io/refactorings/blob/main/lib/refactorings/providers/open-ai/handlers/elementTemplateHandlerDescriptions.json)
* improves prompts
* introduces [seed](https://cookbook.openai.com/examples/reproducible_outputs_with_the_seed_parameter)
* handles non-existing tools
* adds test for OpenAI responses (not executed by default)

### CODE REVIEW

**Comment by philippfromme** [comment]
CI needs to be fixed. OpenAI key is not added even though I added that to the GitHub action. 🤔

**Comment by marstamm** [comment]
Could it be that we are only using `.env` file, so directly set environment variables are not considered when testing? We would need another step which does the `echo 'OPEN_API_KEY='$OPEN_API_KEY > .env` in the CI

**Comment by marstamm** [comment]
I can create a separate PR with the changes

**Comment by philippfromme** [comment]
@marstamm Ready for review ™️

File: lib/refactorings/providers/open-ai/OpenAIClient.js [hunk 1]
```diff
@@ -36,14 +47,28 @@ export default class OpenAIClient {
...
       ],
       model: this._model,
+      seed: this._seed,
+      temperature: this._temperature,
       tool_choice: 'auto',
       tools
     });
 
+    // console.log(JSON.stringify(response, null, 2));
+
```
Comment (marstamm) [inline lib/refactorings/providers/open-ai/OpenAIClient.js]:
```suggestion
```

**Review by marstamm — COMMENTED** [review]
Let's remove this comment and we are good to go 🚀

File: lib/refactorings/providers/open-ai/OpenAIClient.js (hunk 1 above)
Comment (marstamm) [reply lib/refactorings/providers/open-ai/OpenAIClient.js]: Resolved in [a26efd7](https://github.com/bpmn-io/refactorings/pull/6/commits/a26efd77c35a2daf8dd852014da946c296132f47)

**Pull request merged**